Sources/Latte+UniversalLink.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import UIKit
+
+@available(iOS 13.0, *)
+public protocol LatteLinkHandler {
+    func handle(
+        context: UINavigationController,
+        authgear: Authgear,
+        handler: @escaping Latte.ResultHandler<Void>
+    )
+}
+
+@available(iOS 13.0, *)
+public extension Latte {
+    struct ResetLinkHandler: LatteLinkHandler {
+        let customUIEndpoint: String
+        let query: [URLQueryItem]?
+
+        public func handle(
+            context: UINavigationController,
+            authgear: Authgear,
+            handler: @escaping Latte.ResultHandler<Void>
+        ) {
+            Task { await run() }
+            @Sendable @MainActor
+            func run() async {
+                var viewController: LatteViewController?
+                do {
+                    var entryURLComponents = URLComponents(string: customUIEndpoint + "/recovery/reset")!
+                    entryURLComponents.queryItems = query
+                    let entryURL = entryURLComponents.url!
+                    let redirectURI = customUIEndpoint + "/recovery/reset/complete"
```

Comment:
As discussed before, we would set a custom redirect URI (e.g. `latte://recovery-complete`).